packages/prisma/schema.prisma

```diff
@@ -133,6 +133,7 @@ model EventType {
   /// @zod.custom(imports.intervalLimitsType)
   durationLimits                       Json?
   isInstantEvent                       Boolean                   @default(false)
+  instantMeetingExpiryTimeOffset       Int                       @default(90)
```

Comment:
Nitpick here. Are these seconds? minutes? Can we specify in the column name? I know it's already quite long enough. `instantMeetingExpiryTimeOffsetInSeconds`

Reply:
done